_docs/runtime/installation.md

```diff
@@ -1,10 +1,102 @@
+  Note down the external address of the Ingress controller, as you will need this when you install the runtime. 
+
+* **Default `IngressClass`**  
+
+  If you do not have any `IngressClass` resource tagged as the default, add this `ingressclass.kubernetes.io/is-default-class` annotation with the value of `true` to the resource.
+  
+  ```yaml
+  apiVersion: networking.k8s.io/v1
+  kind: IngressClass
+  metadata:
+  annotations:
+    ingressclass.kubernetes.io/is-default-class: "true" 
+  ```
+
+* **Git tokens**  
+
+  For GitHub, you need a Personal Access Token (PAT) for authentication to the Git installation repo, that you will create or select during runtime installation.   
+  To create a Git token, see [Creating a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+
+  > When you create the Git token, set the correct expiration date and scope: 
+   Expiration: Default is `30 days`  
+   Scope: `repo`
+
+#### Runtime installation
+Follow the steps in the runtime installation section, noting the following:
+
+**Runtime name**  
+  Remember that the runtime name must start with a lower-case character, and can include up to 62 lower-case characters and numbers.
+**Mandatory flags**  
+  1. In addition to the runtime name and the runtime installation Git repository, you must pass the Git token to the runtime command. 
+    `--git-token`: The Git token authenticating access to the Git installation repository defined.  
+   
+  When you run `cf runtime install`, you are prompted for:  
+
+  * **Kube context**  
+    Select the Kube context from the list of available contexts.
+  * **Ingress host**
+    The IP address/host name/FQDN of the ingress controller component.
+
+#### Runtime components
+
+**Git repositories**   
+ 
+* Runtime install repo: The installation repo contains three folders: apps, bootstrap and projects, to manage the runtime itself with ArgoCD.  
+* Git source repository: Created with the name `[repo_name]_git-source`. This repo stores manifests to run pipelines aka sources, events, workflow templates.
+
+**Argo CD components**  
+
+* Project, comprising an Argo CD AppProject and an ApplicationSet
+* Installations of the following applications in the project:
+  * Argo CD 
+  * Argo Workflows 
+  * Argo Events
+  * Argo Rollouts
+  
+**Codefresh-specific components**  
+
+* Codefresh Applications in the Argo CD AppProject:  
+  * App-proxy facilitating behind-firewall access to Git 
+  * Git Source entity that references the`[repo_name]_git-source`  
+
+Once the runtime is successfully installed, it is provisioned on the Kubernetes cluster, and displayed on the Runtimes page. 
+
+### What to read next
+[Manage runtimes]({{site.baseurl}}/docs/runtime/monitor-manage-runtimes/)
```

Comment:
Convert this to bullet points. See V1 docs on how it should look.

Reply:
Kostis, I know that these were bullet points  in V1. But list of related topics generally do not have bullets